### PR TITLE
Proposal to add uSTONKS-APR21 as a supported price identifier

### PR DESCRIPTION
**Pull request by smb2796**

### CODE REVIEW

File: UMIPs/ustonks-pid-umip.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+| Discourse Link: To Do   |  
+
+## SUMMARY
+
+The DVM should support price requests for a basket of stocks that represent the top ten tickers with the most comment volume on r/wallstreetbets. These ten stocks will be equally weighted to create an index named uSTONKS.
+
+## MOTIVATION
+
+The DVM currently does not support price identifiers for the creation of indices like uSTONKS.  Financial markets should be universally accessible — unrestrained by censorship, pre-existing financial and social capital, or nationality.  With the uSTONK price identifier synthetic tokens can be created which will allow anybody to obtain exposure to the basket of stocks in a decentralized way.  This will allow unrestricted trading 24 hours a day, 7 days a week.
+
```
Comment (mhairimca) [inline UMIPs/ustonks-pid-umip.md:17]: typo - should read "uSTONKS"

File: UMIPs/ustonks-pid-umip.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
...ortunity for sponsors to mint tokens with less collateral than what they can sell them from in the market. It is important to note that this is an edge case scenario either driven by an irrational change in market expectations or it can be driven by a “fat finger” mistake which is a vulnerability to any market. Even in this edge case we believe the design of the token and the parameters chosen should mitigate risks. The current Expiring Multi Party (EMP) contract requires sponsors to mint tokens with enough collateral to meet the Global Collateral Ratio (GCR) which has stood well above 200% for other contracts. Therefore, assuming the GCR is similar for uSTONKS-MAR, the market would need to first rally at least 100% before potentially being exposed. If the sponsor wishes to withdraw collateral below the GCR they would request a “slow withdrawal” which would subject him to a 2 hour “liveness period” where anybody can liquidate the position if it fell below the required collateral ratio. The combination of the GCR and 2 hour “liveness period” allows the 2 hour TWAP to “catch up” to the market price and would protect from this scenario and deter sponsors from attempting to exploit it.
```
Comment (Tommy1231232) [inline UMIPs/ustonks-pid-umip.md:119]: Could this be reworded a bit? Are you saying a difference in the TWAP and the actual token price? Maybe the word gap is throwing me off haha

File: UMIPs/ustonks-pid-umip.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+## MOTIVATION
+
+The DVM currently does not support price identifiers for the creation of indices like uSTONKS.  Financial markets should be universally accessible — unrestrained by censorship, pre-existing financial and social capital, or nationality.  With the uSTONK price identifier synthetic tokens can be created which will allow anybody to obtain exposure to the basket of stocks in a decentralized way.  This will allow unrestricted trading 24 hours a day, 7 days a week.
+
+## MARKETS & DATA SOURCES
+
+- Markets:
+
+NYSE: GME, AMC, NOK, BB,
```
[the remark this answers is not in the document]
Comment (smb2796) [reply UMIPs/ustonks-pid-umip.md:22]: No they are static. I will make this clearer

File: UMIPs/ustonks-pid-umip.md (hunk 2 above)
Comment (smb2796) [reply UMIPs/ustonks-pid-umip.md:119]: Yep, this is talking about a situation where the TWAP and actual token price are very different

**Review by Tommy1231232 — APPROVED** [review]

**Pull request merged**